compiler/rustc_error_messages/locales/en-US/parse.ftl

```diff
@@ -368,3 +368,9 @@ parse_maybe_fn_typo_with_impl = you might have meant to write `impl` instead of
 
 parse_expected_fn_path_found_fn_keyword = expected identifier, found keyword `fn`
     .suggestion = use `Fn` to refer to the trait
+
+parse_where_clause_before_tuple_struct_body = where clauses are not allowed before tuple struct bodies
+    .label = unexpected where clause
+    .name_label = while parsing this tuple struct
+    .body_label = the struct body
+    .suggestion = consider moving the body before the where clause
```

Comment:
Let's be more forceful with the wording, there's nothing else the user might have meant, this is their only path forward.

```suggestion
    .suggestion = move the body before the where clause
```